packages/sync-service/lib/electric/shapes/querying.ex

```diff
@@ -25,4 +26,11 @@ defmodule Electric.Shapes.Querying do
 
     {query, stream}
   end
+
+  defp columns(root_table, conn) do
+    root_table
+    |> Inspector.load_table_info(conn)
+    |> Enum.map(&~s("#{&1.name}"::text))
```

Comment:
```suggestion
    |> Enum.map(&~s("#{Utils.escape_quotes(&1.name)}"::text))
```

and make `escape_quotes` public on `Utils` - otherwise this fails on a column with `"` in the name. Add  a test as well please!